ci_framework/roles/edpm_build_images/tasks/package.yml

```
@@ -0,0 +1,22 @@
+---
+- name: Package edpm-hardened-uefi image inside container image
+  when:
+    - (cifmw_edpm_build_images_hardened_uefi | bool) or (cifmw_edpm_build_images_all | bool)
+    - cifmw_edpm_build_images_hardened_uefi_package | bool
+  become: "{{ cifmw_edpm_build_images_via_rpm }}"
+  args:
+    chdir: "{{ cifmw_edpm_image_builder_repo_path }}"
+  ansible.builtin.shell:
+    cmd: |
+      buildah bud -f ./Containerfile.image -t edpm-hardened-uefi:{{ cifmw_edpm_build_images_tag }} --logfile {{ cifmw_edpm_build_images_artifacts }}/edpm_images/edpm_hardened_uefi_container_package.log
```

```suggestion
    cmd: >-
      buildah bud -f ./Containerfile.image
      -t edpm-hardened-uefi:{{ cifmw_edpm_build_images_tag }}
      --logfile {{ cifmw_edpm_build_images_artifacts }}/edpm_images/edpm_hardened_uefi_container_package.log
```

Let's make it a bit more readable :)

---

done